src/QueryType/Server/Collections/Query/Action/Create.php

```diff
@@ -294,20 +293,6 @@ public function setAutoAddReplicas(bool $autoAddReplicas): self
         return $this;
     }
 
-    /**
-     * Request ID to track this action which will be processed asynchronously.
-     *
-     * @param string $id
-     *
-     * @return self Provides fluent interface
-     */
-    public function setAsync(string $id): AsyncActionInterface
-    {
-        $this->setOption('async', $id);
-
-        return $this;
-    }
```

Comment:
This overrides the inherited method with the parameter name as the only difference. None of the other classes that inherit it do this.

https://github.com/solariumphp/solarium/blob/3becc5d3f98501de99d4b993f768687d0cee6b28/src/QueryType/Server/Query/Action/AbstractAsyncAction.php#L20-L25